.github/workflows/e2e.yml

```diff
@@ -0,0 +1,92 @@
+name: e2e
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - 'master'
+      - 'v[0-9]*'
+  pull_request:
+    branches:
+      - 'master'
+      - 'v[0-9]*'
+
+jobs:
+  driver:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        driver:
+          - docker
+          - docker-container
+          - kubernetes
+        buildkit:
+          - moby/buildkit:buildx-stable-1
+          - moby/buildkit:master
+        buildkit-cfg:
```

Comment:
better to use a form like `buildkit-cfg: ["buildkit-cfg-true", "buildkit-cfg-false"]`
(Please see the screenshot in my other comment)

Reply:
sgtm